### PR TITLE
provide a tips to fix golangci-lint error

### PR DESCRIPTION
**Pull request by zhzhuang-zju**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Many times, facing golangci-lint errors, such as ```File is not `gci`-ed with --skip-generated -s Standard -s Default -s Prefix```, do not know its meaning, and therefore do not know the specific modification method. At this time, we can automatically fix it through command `golangci-lint run --fix`, and we can understand the reason for the error by viewing the modification points through git diff.
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:
Take #4622 for example，
```shell
(base) ➜  karmada git:(revive) ✗ hack/verify-staticcheck.sh 
Using golangci-lint version:
golangci-lint has version 1.52.2 built with go1.20.2 from da04413a on 2023-03-25T18:11:28Z
test/e2e/lazy_activation_policy_test.go:20: File is not `gci`-ed with --skip-generated -s Standard,Default,Prefix(github.com/karmada-io/karmada) (gci)
        policyv1alpha1 "github.com/karmada-io/karmada/pkg/apis/policy/v1alpha1"
        "github.com/karmada-io/karmada/test/e2e/framework"
        testhelper "github.com/karmada-io/karmada/test/helper"
test/e2e/lazy_activation_policy_test.go:25: File is not `gci`-ed with --skip-generated -s Standard,Default,Prefix(github.com/karmada-io/karmada) (gci)
        "k8s.io/apimachinery/pkg/util/rand"

Please review the above warnings.
If the above warnings make sense, run "golangci-lint run --fix" to auto fix it
If the above warnings do not make sense, feel free to file an issue.

(base) ➜  karmada git:(revive) ✗ golangci-lint run --fix
(base) ➜  karmada git:(revive) ✗ git diff test/e2e/lazy_activation_policy_test.go
diff --git a/test/e2e/lazy_activation_policy_test.go b/test/e2e/lazy_activation_policy_test.go
index bc57439c0..185095a67 100644
--- a/test/e2e/lazy_activation_policy_test.go
+++ b/test/e2e/lazy_activation_policy_test.go
@@ -17,12 +17,13 @@ limitations under the License.
 package e2e
 
 import (
-       policyv1alpha1 "github.com/karmada-io/karmada/pkg/apis/policy/v1alpha1"
-       "github.com/karmada-io/karmada/test/e2e/framework"
-       testhelper "github.com/karmada-io/karmada/test/helper"
        "github.com/onsi/ginkgo/v2"
        appsv1 "k8s.io/api/apps/v1"
        "k8s.io/apimachinery/pkg/util/rand"
+
+       policyv1alpha1 "github.com/karmada-io/karmada/pkg/apis/policy/v1alpha1"
+       "github.com/karmada-io/karmada/test/e2e/framework"
+       testhelper "github.com/karmada-io/karmada/test/helper"
 )
```

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: hack/verify-staticcheck.sh [hunk 1]
```diff
@@ -38,6 +38,7 @@ if golangci-lint run; then
 else
   echo # print one empty line, separate from warning messages.
   echo 'Please review the above warnings.'
+  echo 'If the above warnings make sense, run "golangci-lint run --fix" to auto fix it'
```
Comment (RainbowMango) [inline hack/verify-staticcheck.sh:41]: I guess `golangci-lint run --fix` can not fix all warnings. right?
In that case, the prompt should be more clear.

Comment (zhzhuang-zju) [reply hack/verify-staticcheck.sh:41]: >I guess golangci-lint run --fix can not fix all warnings. right?

I have not found relevant documentation to explain whether the command `golangci-lint run --fix` can solve all warnings. I think we can use it as a modification suggestion. When we do not understand the meaning of the warning, we can deduce the meaning and solution of the warning after running this command, then decide whether to directly use the solution provided by the command.

Comment (zhzhuang-zju) [reply hack/verify-staticcheck.sh:41]: How about
```suggestion
  echo 'Run "golangci-lint run --fix" and "git diff" to view modification suggestions if necessary.'
```

Comment (RainbowMango) [inline hack/verify-staticcheck.sh:41]:
```suggestion
  echo 'Tips: The golangci-lint might help you fix some issues, try with the command "golangci-lint run --fix".'
```
How about this one?

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**